src/seed/seed_db.ts

```diff
@@ -734,22 +739,56 @@ async function reloadAutocompleteData(): Promise<void> {
             const currentDate = new Date();
             if (songsRemoved.length) {
                 logger.info(`${songsRemoved.length} songs removed.`);
-                await sendDebugAlertFileWebhook(
-                    discordDateFormat(currentDate, "f"),
+                const description = `**${songsRemoved.length} songs removed**:\n${songsRemoved.join("\n")}`;
```

Comment:
Do we want to apply some formatting to make it more readable

Reply:
What more formatting is needed? Also more formatting => less entries

Reply:
It gets a bit difficult to read on mobile. Maybe we can put them in a bulleted list ![Screenshot_20240330-221424~2.png](https://github.com/Brainicism/KMQ_Discord/assets/14286913/fc293f42-3fd6-4f3f-932b-952089491ff6)